### ISSUE TITLE
XML Exception tags get ignored

### ISSUE THREAD

**Issue by SilentSin** [Wyamio/Wyam#810]
E.g.

```
/// <summary>...</summary>
/// <exception cref="NullReferenceException">Thrown if ...</exception>
```

This makes Visual Studio show the exception at the bottom of the tooltip, but Wyam doesn't do anything with it. I figure it just needs an "Exceptions" section at the bottom of the member's page.

**Comment by daveaglick** [comment]
Another great feature request - keep them coming!

You may not have noticed, development is progressing quickly in the `vnext-async` branch (soon to be merged into the `vnext` branch once I finish refactoring module execution to be fully async and verify it all works well). That's the main focus right now in preperation for a v3 release. My plan/hope is to do some significant work on the docs capabilities shortly thereafter at which point I'll start running down the bugs and features you've been submitting.

**Comment by SilentSin** [comment]
What's the best way to get that branch? I've just been downloading the latest zip from the Releases page and using a batch file to run it. I'm not really familiar with NuGet or any of the other stuff mentioned on the Obtaining page.

**Comment by daveaglick** [comment]
Trust me, you don't want it yet :). It's not even buildable. Once I've got all the tests back up and running and merge back into the main `vnext` branch, beta packages will start going up on the MyGet NuGet feed at https://www.myget.org/F/wyam/api/v3/index.json. You still won't be able to run those as there's no executable in v3 (see statiqdev/Statiq.Web#668). Eventually the beta of a new app called _Splashdown_ will go out which contains the old blog and docs recipes rolled into a single static generator application (Splashdown is Wyam's answer to Jekyll, Hugo, etc.) That's the point at which you'll want to download and play.

TL;DR: sit tight, good things are coming!
